opium_kernel/test/dune

```diff
@@ -1,4 +1,4 @@
 (tests
- (names core_route middleware_allow_cors)
```

Comment:
This was named core_route to reflect a directory structure that does not exist anymore.